lib/trashed/railclock.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails/railtie'
+require 'trashed/periodic'
+require 'trashed/reporter'
+
+module Trashed
+  class RailClockTie < ::Rails::Railtie
+    config.trashed_periodic = Trashed::PeriodicReporter.new
+
+    initializer 'trashed.periodic' do |app|
+      require 'statsd'
+
+      app.config.trashed_periodic.logger ||= Rails.logger
+
+      Periodic.new config.trashed_periodic
+    end
+  end
+end
```

Comment:
Railties are meant to handle multiple initializers and setup. We're good to merge this into the main Railtie.

Reply:
The only issue there is that I want some ability to only run the Periodic reporter, or the PerRequest reporter (or possibly both). Do you have a suggestion for how to manage that?